.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "crosstool-ng"]
 	path = crosstool-ng
-	url = https://github.com/zephyrproject-rtos/crosstool-ng.git
+	url = https://github.com/adigie/crosstool-ng.git
```

Comment:
Could we make this use the same picolibcpp.specs file and just replace -lstdc++ with -lstdc++-noexcept? Do we need different header paths or something?